script/npm_link_foreman_js.sh

```diff
@@ -6,21 +6,5 @@
 #
 # This script designed to run using `npm run foreman-js:link` in foreman root
 
-set -e
-
-if [[ -z "${FOREMAN_JS_LOCATION}" ]]; then # FOREMAN_JS_LOCATION is empty
-  FOREMAN_JS_LOCATION="../foreman-js"
-  echo "FOREMAN_JS_LOCATION is not defined, using \"${FOREMAN_JS_LOCATION}\" instead"
-elif [ ! -d "${FOREMAN_JS_LOCATION}" ]; then
-  echo "Can't find folder ${FOREMAN_JS_LOCATION}"
-  exit 1
-fi
-
-FOREMAN_JS_LOCATION="../${FOREMAN_JS_LOCATION}"
-FOREMAN_JS_PACKAGES_LOCATION="${FOREMAN_JS_LOCATION}/packages"
-FOREMAN_JS_INSTALL_LOCATION="./node_modules/@theforeman"
-
-set -x
-
-rm -rf $FOREMAN_JS_INSTALL_LOCATION
-ln -s $FOREMAN_JS_PACKAGES_LOCATION $FOREMAN_JS_INSTALL_LOCATION
+echo "This script is deprecated, please use the 'npm run link' in 'foreman-js'"
+echo "See the docs: https://foreman-storybook.surge.sh/?path=/docs/introduction-adding-dependencies--page"
```

Comment:
I'd add `exit 1` to signify it didn't end the way the user expected it to.
```suggestion
echo "See the docs: https://foreman-storybook.surge.sh/?path=/docs/introduction-adding-dependencies--page"
exit 1
```